ReactWindows/ReactNative/Modules/StatusBar/StatusBarModule.cs

```diff
@@ -0,0 +1,167 @@
+﻿using ReactNative.Bridge;
+using ReactNative.UIManager;
+using System;
+using Windows.Foundation.Metadata;
+using Windows.Security.ExchangeActiveSyncProvisioning;
+using Windows.UI;
+using Windows.UI.Core;
+
+namespace ReactNative.Modules.StatusBar
+{
+    /// <summary>
+    /// A module that allows JS to set statusbar properties.
+    /// </summary>
+    class StatusBarModule : NativeModuleBase
+    {
+        private bool _hidden;
+        private bool _translucent;
+        private Color? _color;
+
+        /// <summary>
+        /// Instantiates the <see cref="StatusBarModule"/>.
+        /// </summary>
+        internal StatusBarModule()
+        {
+            //Mobile customization
+            if (ApiInformation.IsTypePresent("Windows.UI.ViewManagement.StatusBar"))
+            {

```

Comment:
Could we create an interface for the status bar and do this check exactly once?  We could even create a "NopStatusBar" type that we use in cases where having a status bar doesn't make sense (or that emits warnings when status bar operations occur on it).  This would also make the module inherently more testable (we could create a MockStatusBar and assert operations on that).